java/gae-flexible-helloworld/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,24 +1,19 @@
-<web-app xmlns="http://java.sun.com/xml/ns/javaee" version="2.5">
-<!-- [START_EXCLUDE] -->
-<!--
-Copyright 2015 Google Inc. All Rights Reserved.
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--->
-<!-- [END_EXCLUDE] -->
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Missing license header.

Though I believe it is not strictly required on purely config files.